cmd/stats/stats_test.go

```diff
@@ -0,0 +1,395 @@
+package stats
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+func TestParseDateFromArg(t *testing.T) {
+	// 2024-12-06 17:36:20.000000 -0700
+	now := time.Date(2024, 12, 6, 17, 36, 20, 0, time.Now().Location())
```

Comment:
These tests would benefit from having `todayAtMidnight` and `todayBeforeMidnight` variables.